assets/js/applicationinsights.js

```diff
@@ -1,5 +1,5 @@
 /* global Microsoft */
-window.applicationInsights = (function () {
+window.applicationInsights = (() => {
```

Comment:
The new linter didn't like anonymous functions like we had them previously, so I've changed them to ES5 function expressions